datafusion/physical-expr/src/aggregate/count.rs

```diff
@@ -331,123 +331,3 @@ impl Accumulator for CountAccumulator {
         std::mem::size_of_val(self)
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use crate::expressions::tests::aggregate;
-    use crate::expressions::{col, lit};
-    use crate::generic_test_op;
```

Comment:
I plan to move others test as well, so we can move the functions easily without importing test-specific things.